CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Contributing
+
+Feel free to ask any questions in issues or on this [email](mailto:support@mesoform.com).
+
+### Pull Request Process
+
+Before sending pull request, run `black .` and commit changes, if they are present after reformatting.
```

Comment:
Could be worth specifying where `black` comes from, with no experience with the tool its not obvious, I presume it is the python formatting tool

Reply:
it is. Good spot.